roles/jd-client/src/lib/mod.rs

```diff
@@ -368,8 +368,9 @@ impl JobDeclaratorClient {
         .await;
     }
 
-    pub fn shutdown(&self) -> Arc<Notify> {
-        self.shutdown.clone()
+    #[allow(dead_code)]
+    pub fn shutdown(&self) {
```

Comment:
Could you please document this function? it would be a good practice to start using the `roles` as well